proposals/utils/stepper_helpers.py

```diff
@@ -129,16 +148,14 @@ def is_current(self, request):
         """
         return False
 
-    def css_classes(self):
-        css_classes = super().css_classes()
+    def get_css_classes(self):
         child_errors = []
         for child in self.children:
             child_errors += child.get_errors()
         if child_errors != []:
-            css_classes += "incomplete"
+            self.css_classes += "incomplete"
```

Comment:
There isn't enough consistency in where the space goes. I think we might better avoid the issue entirely and provide a `list` of css classes, and only turn them into a string when needed for HTML.